ospd/ospd.py

```diff
@@ -602,20 +602,26 @@ def sort_host_finished(
         )
 
     def set_scan_progress_batch(
-        self, scan_id: str, host_progress: Dict[str, int]
+        self, scan_id: str, host_progress: Dict[str, Union[float, int]]
     ):
         self.scan_collection.set_host_progress(scan_id, host_progress)
 
         scan_progress = self.calculate_progress(scan_id)
         self.scan_collection.set_progress(scan_id, scan_progress)
 
     def set_scan_host_progress(
-        self, scan_id: str, host: str = None, progress: int = None,
+        self,
+        scan_id: str,
+        host: str = None,
+        progress: Union[float, int] = None,
```

Comment:
Why should be a float still allowed?

Reply:
This comes from the wrapper. It depends on the wrapper implementation. Currently ospd-openvas sends a float value

Reply:
Can we fix that?